.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Documentation Build Check
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
```

Comment:
don't think it matters much, but I think these actions have more recent versions

Reply:
I updated them, thanks